### PR TITLE
fix(VFileInput): add hideInput prop

### LINKED ISSUE #17142: [Bug Report][3.1.14] <v-file-input :hide-input="true" /> can not hide to input
### Environment
**Vuetify Version:** 3.1.14
**Vue Version:** 3.2.47
**Browsers:** Edge 112.0.1722.39
**OS:** Windows 10

### Steps to reproduce
```
    <v-form style="width: 50%; margin: 0 auto">
      <v-file-input
        class="fileInput"
        label="Select a image"
        accept="image/*"
        :hide-input="true"
        :clearable="false"
        prepend-icon="mdi-camera"
      >
      </v-file-input>
```
both setting `:hide-input="true"` or `hide-input="true"` are not works at all.

### Expected Behavior
hide input field

### Actual Behavior
can not hide the input field

### Reproduction Link
[https://codesandbox.io/s/wonderful-glade-m06big?file=/src/components/HelloWorld.vue](https://codesandbox.io/s/wonderful-glade-m06big?file=/src/components/HelloWorld.vue)

image

### PR DESCRIPTION
**Pull request by Reidemption**
My attempt at adding the hideInput prop back in for Vuetify 3. Please let me know if it requires more changes. :^)

## Description

I added a hideInput prop and toggled the styling to `display: 'none'` if the prop is set to true. I'm sure there are better ways to do this, but this seemed the simplest. I made this change because it was previously an option in v2, but it currently is not in v3.

resolves #17142
## Markup:

```vue
<template>
  <div class="hello">
    <v-form style="width: 50%; margin: 0 auto">
      <v-file-input
        class="fileInput"
        label="Select a image"
        accept="image/*"
        :hide-input="true"
        :clearable="false"
        :counter="false"
      />
    </v-form>
  </div>
</template>

<script>
  export default {
    name: 'HelloWorld',
  }
</script>

<!-- Add "scoped" attribute to limit CSS to this component only -->
<style scoped lang="scss">
  .hello {
    display: flex;
    justify-content: center;
    align-items: center;
    height: 100%;
  }
  .fileInput {
    width: 100%;
  }
</style>

```

### CODE REVIEW

**Comment by landynfrancis** [comment]
Does anyone have an idea when this PR will be merged in? I need the Hide Input prop for my VFileInput.

File: packages/vuetify/src/components/VFileInput/VFileInput.tsx [hunk 1]
```diff
@@ -193,6 +195,7 @@ export const VFileInput = genericComponent<VFileInputSlots>()({
                 disabled={ isDisabled.value }
                 focused={ isFocused.value }
                 error={ isValid.value === false }
+                style={ !hasInput ? { display: 'none' } : props.style }
```
Comment (yuwu9145) [inline packages/vuetify/src/components/VFileInput/VFileInput.tsx:198]: consider to use existing utility class `d-none`

Comment (johnleider) [reply packages/vuetify/src/components/VFileInput/VFileInput.tsx:198]: Utility classes should not be used in components.

**Review by johnleider — CHANGES_REQUESTED** [review]
Instead of manually applying a style, use a class and then "display: none" in VFileInput.sass.

**Comment by johnleider** [comment]
> Hey all, any idea when this will get merged in? Been looking forward to this change for a little bit now.

I left feedback for the author.

**Comment by landynfrancis** [comment]
@Reidemption

**Review by johnleider — APPROVED** [review]

**Pull request merged**